<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramWindow</class>
 <widget class="QMainWindow" name="ProgramWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1641</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgramWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FPS: 20</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>630</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2D to 3D</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1641</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Закраска</string>
     </property>
     <addaction name="settings_fill_phong"/>
     <addaction name="settings_fill_gouraud"/>
     <addaction name="settings_fill_mono"/>
     <addaction name="separator"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="settings_poly"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Ввод формул</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Поддерживаемые соединения</string>
   </property>
  </action>
  <action name="settings_fill_menu">
   <property name="text">
    <string>Закраска</string>
   </property>
   <property name="toolTip">
    <string>Меню выбора способа закраски</string>
   </property>
  </action>
  <action name="settings_fill_phong">
   <property name="text">
    <string>Каркасная модель</string>
   </property>
   <property name="toolTip">
    <string>Установка закраски методом фонга</string>
   </property>
  </action>
  <action name="settings_fill_gouraud">
   <property name="text">
    <string>Гуро</string>
   </property>
   <property name="toolTip">
    <string>Установка закраски методом Гуро</string>
   </property>
  </action>
  <action name="settings_fill_mono">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Однотонная</string>
   </property>
   <property name="toolTip">
    <string>Установка однотонной закраски</string>
   </property>
  </action>
  <action name="settings_poly">
   <property name="text">
    <string>Графика</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
